scss/_objects.grid.scss

```diff
@@ -4,47 +4,63 @@
     Grid system objects, works together with utility width classes
 */
 
+
+// Max width container
 .o-container {
     margin-left: auto;
     margin-right: auto;
     max-width: $grid-max-width;
 }
 
-// No gutters
 
 .o-grid {
-    @include clearfix;
+    display: flex;
+    align-items: stretch;
+    flex-direction: row;
+    flex-wrap: wrap;
 }
 
-// Default gutter
+    // 100% width by default
+    .o-grid__item {
+        width: 100%;
```

Comment:
@Dietr should we use `flex: 1;` in stead of `width: 100%;`? I know they do different things but I’m not sure about the default expected behaviour.